web-api/openapi/front_office.yaml

```
@@ -165,6 +200,40 @@ components:
           type: number
         properties:
           $ref: './schemas/document.yml#/DocumentProperties'
+    StatelessPersonalizedDocumentsRequest:
+      type: object
+      required: [history]
+      properties:
+        history:
+          $ref: './schemas/document.yml#/History'
+        count:
+          $ref: '#/components/schemas/Count'
+        published_after:
+          $ref: './schemas/time.yml#/PublicationDate'
+    StatelessPersonalizedDocumentsResponse:
+      type: object
+      required: [documents]
+      properties:
+        warnings:
+          type: array
+          minItems: 0
+          maxItems: 100
+          items:
+            type: string
+        documents:
+          type: array
+          minItems: 0
+          maxItems: 100
+          items:
+            $ref: '#/components/schemas/PersonalizedDocumentData'
+      example:
+        warnings:
+          - "historic document with id \"document-id-1\" does not exist"
```

:art: maybe like this to align it with the doc id example two lines below
```suggestion
          - "historic document with id \"document_id1\" does not exist"
```